openslides/core/models.py

```diff
@@ -243,3 +273,61 @@ class Meta:
     def get_session_store_class(cls):
         from .session_backend import SessionStore
         return SessionStore
+
+
+class Message(RESTModelMixin, models.Model):
```

Comment:
I think the problem is that messages and countdowns are stable (they have to be stable!), but if one wants to remove a message, he has to give the uuid. My idea was to give one message always the same uuid (generated at the moment of creation), so if the message is projected on two projectors. it has the same uuid on both projectors. For writing in a projectors config and removing from there the uuid is necessary.